### PR TITLE
Adding aprs.fi

### PR DESCRIPTION
**Pull request by Luceoria**

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1065,6 +1065,19 @@ INVERT
...
 
+aprs.fi
+*.aprs.fi
+
+INVERT
+.gm-style
+.gm-style-iw-a
+
+CSS
+#panel {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1076]: image
The panel looks good to me?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1065,6 +1065,19 @@ INVERT
 
 ================================
 
+aprs.fi
+*.aprs.fi
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1069]:
```suggestion
```

No need to specify the sub-domain.

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -1065,6 +1065,19 @@ INVERT
...
+
+INVERT
+.gm-style
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1072]:
```suggestion
.gm-style > :first-child
```

File: src/config/dynamic-theme-fixes.config [hunk 4]
```diff
@@ -1065,6 +1065,19 @@ INVERT
...
+.gm-style-iw-a
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1073]: Hmm I don't seem to find this selector.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Luceoria) [reply src/config/dynamic-theme-fixes.config:1075]: If you take a closer look, "Login" href just above the cookies disclaimer is almost not visible with current background color.

File: src/config/dynamic-theme-fixes.config (hunk 4 above)
Comment (Luceoria) [reply src/config/dynamic-theme-fixes.config:1072]: This one is displayed when any node is clicked on the map:
image

**Review by Gusted — APPROVED** [review]
Thank you!

File: src/config/dynamic-theme-fixes.config [hunk 5]
```diff
@@ -1065,6 +1065,18 @@ INVERT
...
+INVERT
+.gm-style > :first-child
+.gm-style-iw-a
+
+CSS
+#panel {
+    background-image: none !important;
+}
+
+================================
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1079]:
```suggestion
================================

```

**Pull request merged**